mappings/net/minecraft/client/gui/widget/GameOptionSliderWidget.mapping

```diff
@@ -1,5 +1,8 @@
 CLASS cxf net/minecraft/client/gui/widget/GameOptionSliderWidget
 	FIELD a option Lcvq;
+	METHOD <init> (Lcvo;IIIILcvq;)V
+		ARG 1 options
+		ARG 6 option
```

Comment:
The difference between the two is not immediately clear to me.

Reply:
options is the instance of GameOptions, option is the actual option itself.
Not sure what I should name them

Reply:
Though it's more verbose, `gameOptions` should point out the difference better. Not sure if there might be a better solution.